apps/sharedtest/test/unit/template_test.js

```diff
@@ -61,8 +61,10 @@ suite('Template', function() {
 
   suite('interpolate', function() {
     var html = document.createElement('div');
+    var htmlNoIntrpltn = document.createElement('div');
     var css = document.createElement('div');
     html.appendChild(document.createComment('<span>${str}</span>'));
+    htmlNoIntrpltn.appendChild(document.createComment('<span>Nothing</span>'));
     css.appendChild(document.createComment('#foo { height: ${height}px; }'));
```

Comment:
It's quite disturbing all this is not in a setup() (which means they would be renegerated again for each test, thus we're sure we have a clean state). I know it was like this before already so if you prefer to keep it like this it's also fine to me.